src/jit/MemoryInl.h

```diff
@@ -709,11 +709,7 @@ static void emitStore(sljit_compiler* compiler, Instruction* instr)
             addr.loadArg.argw = 0;
         }
 
-#if (defined SLJIT_FPU_UNALIGNED && SLJIT_FPU_UNALIGNED)
         sljit_emit_fop1(compiler, opcode, addr.memArg.arg, addr.memArg.argw, addr.loadArg.arg, addr.loadArg.argw);
-#else /* SLJIT_FPU_UNALIGNED */
-        sljit_emit_fmem(compiler, opcode | SLJIT_MEM_UNALIGNED, addr.loadArg.arg, addr.memArg.arg, addr.memArg.argw);
-#endif /* SLJIT_FPU_UNALIGNED */
```

Comment:
Is this an optimization?

Reply:
This is a fix. The previous implementation was bad.